mods/ts/rules/defaults.yaml

```diff
@@ -609,6 +609,17 @@
 	Tooltip:
 		Name: Palette
 
+^Tunnel:
+	AlwaysVisible:
+	Immobile:
```

Comment:
You can then also make the inner-most row of cells impassible, which will stop units from pathing through the tunnel to the ground above.